common/decrees/00_decree.txt

```diff
@@ -525,6 +522,8 @@ decree_encourage_agricultural_industry = {
 		building_group_bg_agriculture_throughput_add = 0.2
 		building_group_bg_plantations_throughput_add = 0.2
 		building_group_bg_ranching_throughput_add = 0.2
+		country_farmers_pol_str_mult = 0.01
+		country_aristocrats_pol_str_mult = 0.01
```

Comment:
This 1% is too insignificant to add in my opinion, I would suggest making it relevent (like 5%) or removing it. To prevent cluttering.